test/e2e/common/setup-kubestellar.sh

```diff
@@ -66,6 +65,22 @@ make install-local-chart KUBE_CONTEXT=kind-kubeflex
 cd -
 echo "wds1 created."
 
+:
+: -------------------------------------------------------------------------
+: Run OCM transport controller executable
+:
+cd "${SRC_DIR}/../../.." ## go up to KubeStellar directory
+KUBESTELLAR_DIR="$(pwd)"
+cd ocm-transport-plugin-0.1.0-rc1
+echo "replace github.com/kubestellar/kubestellar => ${KUBESTELLAR_DIR}/" >> go.mod
+make build
+./bin/ocm-transport-plugin --transport-context imbs1 --wds-context wds1 --wds-name wds1 >/dev/null &
```

Comment:
Do you implicitly assume a KUBECONFIG is set and points to KF ?

Reply:
this is not something I've introduced. this is how the test is written today in main.
it assumes a single KUBECONFIG and uses context to switch between different clusters.
see here for example - https://github.com/kubestellar/kubestellar/blob/b2382485dc3da5c4ca5d8692940c470bf3d5ca00/test/e2e/common/setup-kubestellar.sh#L51

I just re-used whatever exists in current tests

Reply:
@ezrasilvera: The assumption is that there is a kubeconfig file that standard kube clients will use (for the standard reasons: either being named in `$KUBECONFIG` or located at `~/.kube/config`) and it has two specific _contexts_ within it for the ITS and WDS.